### DISCUSSION TITLE
Question on large albums

### DISCUSSION THREAD

**Discussion by haivala** [LycheeOrg/Lychee#2287] [Q&A] [answered]
I had an project. Result was 80k pictures and they take about 8G of space. 720p images. 

What is the recommended way to handle the upload and (as I remember someone having problems with huge albums) is there recommended way to split the pics in albums or should it work as expected now?

I would prefer zip upload through browser (temporarily allowing huge file) but If I use that how it is handled? for example: If there is folder in the zip does it become a sub album? or does it even work? 

Tips?

**Comment by ildyria** [comment, marked as answer]
First this.

> I would prefer zip upload through browser (temporarily allowing huge file) but If I use that how it is handled? for example: If there is folder in the zip does it become a sub album? or does it even work?

The good news is.. it is in the working: https://github.com/LycheeOrg/Lychee/pull/2229 (still need more work on it).
The way it **will** work is:
1. zip is put on server (I don't remember where in `/storage`
2. zip is extracted to `/storage/extract-jobs/zip-file-name`
3. `/storage/extract-jobs/zip-file-name` is imported with an import queue.
4. profit

The upload by chunk should take care of your 8GB zip file **once #2140 is merged**.
Now the problem is that #2229 does not support sub folders yet (but that is in the working).

## TL;DR:
> What is the recommended way to handle the upload and (as I remember someone having problems with huge albums) is there recommended way to split the pics in albums or should it work as expected now?

For now I would recommend to upload the pictures via ftp into `public/uploads/import/` and then use command line:
`php artisan lychee:sync --delete_imported=1` to take care of that (the tree structure of your folders will be converted into albums).

**Reply by haivala** [reply]
This should be marked as answered.

**Comment by haivala** [comment]
Thank you! That is what I thought.

Should this be added (in some form) in the faq?

**Comment by haivala** [comment]
if I use the sync command is there an option to change owner of the album to user? or is it automatically the admin?

**Reply by ildyria** [reply]
You can add `--owner_id=1` where `1` is the user_id of the owner, by default it is admin.
That being said, using the v5, you can go into the album details and change the owner easily.

**Comment by ildyria** [comment]
Done for SE users. :)
